pyproject.toml

```diff
@@ -3,14 +3,12 @@ requires = ["setuptools", "wheel"]
 
 [project]
 name = "awpy"
-authors = [
-    {name = "Peter Xenopoulos", email = "xenopoulos@nyu.edu"},
-]
+authors = [{ name = "Peter Xenopoulos", email = "xenopoulos@nyu.edu" }]
```

Comment:
add yourself as an author, if you are comfortable with being publicly listed

Reply:
I will, thanks!